google/cloud/pubsub/mocks/mock_pull_ack_handler.h

```diff
@@ -34,7 +35,9 @@ class MockPullAckHandler : public pubsub::PullAckHandler::Impl {
  public:
   MOCK_METHOD(future<Status>, ack, (), (override));
   MOCK_METHOD(future<Status>, nack, (), (override));
+  MOCK_METHOD(std::string, ack_id, (), (const, override));
   MOCK_METHOD(std::int32_t, delivery_attempt, (), (const, override));
+  MOCK_METHOD(pubsub::Subscription, subscription, (), (const, override));
```

Comment:
Nit: Is there any reason to declare these in a different order than the actual functions?

Reply:
No I was trying to make it alphabetical, but did not do so in two places.

Reply:
done